pkg/dagger.io/dagger/core/fs.cue

```diff
@@ -92,6 +92,29 @@ import "dagger.io/dagger"
 	dest: string
 }
 
+// Remove file or directory from a filesystem tree
+#RmFile: {
+	$dagger: task: _name: "RmFile"
+
+	// Input filesystem tree
+	input: dagger.#FS
+
+	// Path to delete (handle wildcard)
+	// (e.g. /file.txt or /*.txt)
+	path: string
+
+	// Allow wildcard selection
+	// Default to: true
+	allowWildcard: *true | bool
+
+	// Do not report error when file do not exists
```

Comment:
```suggestion
	// Do not report error when path does not exist
```

Reply:
Since `allowNotFound` is not supported anymore, this change can not be applied.
See commit : https://github.com/dagger/dagger/pull/2422/commits/e39926bccdb5ab6328c32cfc67f34574111c1d50